src/con_duct/suite/ls.py

```diff
@@ -153,9 +153,37 @@ def pyout_ls(run_data_list: List[OrderedDict[str, Any]]) -> None:
     if pyout is None:
         raise RuntimeError("pyout is required for this output format.")
 
+    E = dict(
+        color=dict(
+            re_lookup=[
+                ["^0$", "green"],  # if exactly "0", then green
+                [".*", "red"],  # anything else gets red
+            ]
+        )
+    )
+    N = dict(
+        color=dict(
+            re_lookup=[
+                [f"^{SummaryFormatter.NONE}", "red"],  # if starts with NONE
+                [".*", "green"],
+            ]
+        )
+    )
+    S = N  # S is humansize, that conversion has already happened, the rest is identical to N
     with pyout.Tabular(
         style=dict(
             header_=dict(bold=True, transform=str.upper),
+            average_pcpu=N,
+            average_pmem=N,
+            average_rss=S,
+            average_vsz=S,
+            end_time=N,
+            exit_code=E,
+            memory_total=S,
+            peak_pcpu=N,
+            peak_rss=S,
+            peak_vsz=S,
+            start_time=N,
```

Comment:
per agreement -- replace with a little `**{}` comprehension over the other style to instruct which one to use. And may be for that replace individual variables with a `modifiers = {'S': ..., 'N': ...}` so you could just take the "modifier" character using regex from the original format.

Reply:
and here keep it empty dict if no colors